modules/va_facilities/README.v1.yml

```diff
@@ -0,0 +1,599 @@
+openapi: '3.0.0'
+info:
+  version: 1.0.0
+  title: VA Facilities
+  description: |
+
+    ## Background
+
+    This API provides information about physical VA facilities. Information available includes
+    geographic location, address, phone, hours of operation, and available services. 
+
+    VA operates several different types of facilities, the types represented in this API include:
+    - Health facilities
+    - Benefits facilities
+    - Cemeteries
+    - Vet Centers
+
+    ## Design
+
+    ### Authorization
+
+    API requests are authorized through a symmetric API token, provided in an HTTP header with name "apikey".
+
+    ### Response Formats
+
+    Clients may request several response formats by setting the `Accept` header.
+    - application/json - The default JSON response format complies with JSON API. This media type is *not* available for bulk requests using the `/facilities/all` endpoint. It will return `406 Not Acceptable`. .
+    - application/vnd.geo+json - GeoJSON-compliant format, representing each facility as a Feature with a Point geometry.
+    - text/csv - Available for the bulk download operation only. Some structured fields are omitted from the CSV response.
+
+    ### Response Elements
+
+    Some data elements within the response are only present for facilities of a given type:
+    - The patient satisfaction scores contained in the `satisfaction` element are only applicable
+      to VA health facilities.
+    - The patient wait time values contained in the `wait_times` element are only applicable to
+      VA health facilities.
+    - The list of available services in the `services` element is only applicable to VA health and
+      benefits facilities.
+
+    ## Reference
+
+    - [Raw VA Facilities Open API Spec](http://dev-api.va.gov/services/va_facilities/docs/v1/api)
+    - [GeoJSON Format](https://tools.ietf.org/html/rfc7946)
+    - [JSON API Format](http://jsonapi.org/format/)
+
+  termsOfService: ''
+  contact:
+    name: developer.va.gov
+tags:
+  - name: facilities
+    description: VA Facilities API
+servers:
+  - url: https://dev-api.va.gov/services/va_facilities/{version}
+    description: VA.gov API development environment
+    variables:
+      version:
+        default: v1
+  - url: https://staging-api.va.gov/services/va_facilities/{version}
+    description: VA.gov API staging environment
+    variables:
+      version:
+        default: v1
+  - url: https://api.va.gov/services/va_facilities/{version}
+    description: VA.gov API production environment
+    variables:
+      version:
+        default: v1
+paths:
+  /nearby:
+    get:
+      tags:
+        - facilities
+      summary: Query facilities based on drive time from a specified point
+      description: |
+        Retrieve all facilities contained within an isochrone based on a the passed `lat`, `lng`, and `drivetime` parameters.
```

Comment:
any thoughts on drivetime vs drive_time as the param? we have per_page as a param so didn't know if we should match them up

Reply:
I was avoiding the underscore (mostly because I refuse to believe this usage of "drivetime" is actually two words), but you're right: since we are already using it for `per_page` it should probably be `drive_time`.